README.md

```diff
@@ -8,7 +8,7 @@
 
   Here's a gif of a new game being started.
 
-  ![Gif of game screen](https://media2.giphy.com/media/v1.Y2lkPTc5MGI3NjExZjgycTdlazZzbHJ4MnljZm1jOW9rOGY0Y3NydXlreWl4YTNlc212NSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/DG0aoSpfx92NYjNOKO/source.gif)
+  ![Gif of game screen](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExcHUzdjYzMzhpanVuNnRqZmVkMW02cTVzY2czc203NnBycnpxczBlYyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/31yA5QunTkbB7ktMLr/giphy.gif)
```

Comment:
What problems were there for the original file? Trying to look for some, I only found some weird lines that are not too noticeable.

On the other hand, the new upload to GIPHY is way more grainy than the clear original.

Reply:
The problem I faced was that the original file is cut, so it will show the game from "run" to the point you "enter your name" and then it replays, because of that I had to use a 3rd party app to create the GIF file image in which is possible to specify the duration of the GIF image.
If the quality of GIF is not acceptable, I try to find another service or resolve the problem in another way.

Reply:
I uploaded another version of the GIF file